### PR TITLE
First release PR

### PR DESCRIPTION
**Pull request by BorisDog**

### CODE REVIEW

File: src/MongoDB.Analyzer/Core/Builders/BuildersAnalyzer.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        }
+
+        return telemetry.ExpressionsFound > 0;
+    }
+
+    private static AnalysisStats ReportMqlOrInvalidExpressions(MongoAnalyzerContext context, ExpressionsAnalysis buildersAnalysis)
+    {
+        var semanticContext = context.SemanticModelAnalysisContext;
+        if (buildersAnalysis.AnalysisNodeContexts.EmptyOrNull())
+            return AnalysisStats.Empty;
```
Comment (JamesKovacs) [inline src/MongoDB.Analyzer/Core/Builders/BuildersAnalyzer.cs:55]: Always include braces. Same elsewhere.

**Review by JamesKovacs — APPROVED** [review]
This PR incorporates my feedback from the previous PR. I realize you incorporated the feedback in this branch and PR because the same feedback was applicable to new code that you had written. So we can ignore the oddity of the split PRs.

One additional thing I noted is that there are a few places where you have single-line `if` statements without parens. According to our team coding style, we should include the parens. You can add these where necessary and then merge.

One other point of feedback. Since this is a new repo, the main branch should be called `main` and not `master`. Github has an automated renamed from `master`->`main` that you can use to fix this.

LGTM.

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Comment by BorisDog** [comment]
Merged

**Pull request closed without merging**